e2e/command_test.go

```diff
@@ -599,6 +615,83 @@ func TestE2E_CreateTaskCommand_NoTaskFileProvided(t *testing.T) {
 	}
 }
 
+// TestE2E_CreateDeleteCreateTrigger tests that after creating a task, then deleting
+// the task, and then finally re-creating the task, the task will trigger
+func TestE2E_CreateDeleteCreateTrigger(t *testing.T) {
+	t.Parallel()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "create_cmd")
+
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+	cts := ctsSetup(t, srv, tempDir, dbTask())
+
+	// Write task config file
+	var taskConfig hclConfig
+	taskName := "new-task"
+	inputTask := fmt.Sprintf(`
+task {
+  name           = "%s"
+  description    = "Creates a new task"
+  module         = "./test_modules/local_instances_file"
+  providers      = ["local"]
+  services       = ["api"]
+  enabled = true
+}`, taskName)
+	taskConfig = taskConfig.appendString(inputTask)
+	taskFilePath := filepath.Join(tempDir, "task.hcl")
+	taskConfig.write(t, taskFilePath)
+
+	// Create task
+	subcmdCreate := []string{"task", "create",
+		fmt.Sprintf("-%s=%s", command.FlagHTTPAddr, cts.FullAddress()),
+	}
+	subcmdCreate = append(subcmdCreate, fmt.Sprintf("--task-file=%s", taskFilePath))
```

Comment:
nit: rather than appending `--task-file` after `subcmdCreate` is declared & initialized, could it be included when declared & initialized?